packages/web-components/src/components/va-need-help/va-need-help.tsx

```diff
@@ -0,0 +1,29 @@
+import { Component, Host, h } from '@stencil/core';
+
+/**
+ * @componentName Need Help?
```

Comment:
It might be the `?` that is making the maturity category mapping to fail. We might want to remove that character from here as well as in the component's Storybook file and rebuild.

Reply:
Yeah, I thought that might be it and tried toggling it but it turned out to be the path that made the maturity dot show up.